react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -119,7 +118,53 @@ const enableLassoArgs = {
         {
             "@@type": "LassoLayer",
             visible: true,
-            data: "@@#resources.wellsData",
         },
     ],
 };
+
+export const lassoSelectionWithCallback: ComponentStory<
+    typeof DeckGLMap
+> = () => {
+    const [data, setData] = React.useState<string[]>([]);
+    const getSelectedWellsDataCallBack = React.useCallback(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (pickingInfos: any[]) => {
+            const selectedWells = pickingInfos
+                .map((item) => item.object)
+                .filter((item) => item.type === "Feature")
+                .map((item) => item.properties["name"]) as string[];
+            setData(selectedWells);
+        },
+        []
+    );
+    const lassoArgsWithSelectedWellsDataCallback: Record<string, unknown> = {
+        ...disableLassoArgs,
+        layers: [
+            {
+                "@@type": "WellsLayer",
```

Comment:
Is it necessary to use the JSON format for specifying layers here? Could you use the typed classes, ie. `[new WellsLayer({data: ...}), new LassoLayer({...})]`?

Reply:
hmmm I updated and it seems it does not like the typed classes format..
![image](https://user-images.githubusercontent.com/39239025/205269751-39b99741-9766-4ee3-b068-ad04c344ce39.png)

Reply:
Ok, never mind then. Suggest that we fix this in the future.